application/src/main/java/org/opentripplanner/graph_builder/module/DirectTransferGenerator.java

```diff
@@ -117,47 +142,32 @@ public void buildGraph() {
         LOG.debug("Linking stop '{}' {}", stop, ts0);
 
         // Calculate default transfers.
-        for (RouteRequest transferProfile : transferRequests) {
+        for (RouteRequest transferProfile : transferConfiguration.defaultTransferRequests) {
```

Comment:
I think it's better if you use the record fields through the getters instead of accessing the fields directly.

Reply:
This has now been changed